encrypted-media-respec.html

```diff
@@ -1811,7 +1810,7 @@
                       <p class="note">In other words, do not create a session if a non-closed persistent session already exists for this <var>sanitized session ID</var> in any browsing context.</p>
                     </li>
                     <li><p>Load the <var>session data</var>.</p></li>
-                    <li><p>If the <var>session data</var> indicates an expiration time for the session, let <var>expiration time</var> be the expiration time in milliseconds since 01 January 1970 UTC.</p></li>
+                    <li><p>If the <var>session data</var> indicates an expiration time for the session, let <var>expiration time</var> be that expiration time.</p></li>
```

Comment:
This one was fixed (removing millisecond); it just doesn't refer to the definition of Time. I believe your suggestion is consistent with the first option I suggested, and LGTM. The one difference is that you changed "that" back to "the." I'm not sure if that was intentional; "that" seemed to more precisely reference the one we already referenced. This style also simplifies line 2328.